src/support.sch

```diff
@@ -16,10 +16,40 @@
         <rule context="*">
             <assert test="
                 name()='article'
+                or name()='article-meta'
+                or name()='article-title'
+                or name()='front'
+                or name()='title-group'
             " role="warn">
                 &lt;<name/>&gt; is ignored.
             </assert>
         </rule>
     </pattern>
 
+    <pattern id="article-title-parent">
+        <rule context="article-title[parent::*]">
+            <let name="parent" value="name(..)"/>
+            <assert test="$parent='title-group'" role="warn">
+                &lt;<name/>&gt; in &lt;<value-of select="$parent"/>&gt; is ignored.
```

Comment:
@FAtherden-eLife Is there a way to simplify these? There's going to be a lot of checking that elements have certain parents.

Reply:
@thewilkybarkid not that I'm aware of in XSL 1 I'm afraid.